echopop/nwfsc_feat/utils.py

```diff
@@ -666,3 +721,10 @@ class TSLRegressionParameters(InputModel, title="TS-length regression parameters
 
     slope: float = Field(allow_inf_nan=False)
     intercept: float = Field(allow_inf_nan=False)
+
+
+def roundn(n):
+    """
+    Rounding function to avoid bankers rounding
+    """
+    return n.apply(lambda x: np.floor(x + 0.5) if x >= 0 else np.ceil(x - 0.5))
```

Comment:
What is the net effect you want to achieve here? The condition tests for if x>=0, so it does not seem to be a general rounding function but something that specifically handles values above/below 0. Depending on the goal, maybe change the function name?